include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -3038,6 +3038,10 @@ struct OrtApi {
    */
   ORT_API2_STATUS(HasValue, _In_ const OrtValue* value, _Out_ int* out);
 
+  ORT_API2_STATUS(GetValueMemoryInfo, _In_ const OrtValue* value, _Outptr_ const OrtMemoryInfo** memory_info);
+
+  ORT_API2_STATUS(GetExecutionProviderApi, _In_ const char* provider_name, uint32_t version, const void** provider_interface);
```

Comment:
```suggestion
  ORT_API2_STATUS(GetExecutionProviderApi, _In_ const char* provider_name, uint32_t version, const void** provider_api);
```
All valid provider names, rationale for adding this API, what it does, etc. need to be documented here.
What value of version should be used here? Is this different from ORT_API_VERSION?

Reply:
Same as ORT_API_VERSION